README.md

```diff
@@ -42,54 +42,54 @@ Below is a table illustrating the main differences and our recommendations:
 | **Built-in File Storage**     |                           | ✅                  |
 
 ### 1.2 Why is ZHE3 important?
-- Lay the groundwork for near-parity with ZenHub Cloud release cycle ✅
+- Lay the groundwork for near-parity with Zenhub Cloud release cycle ✅
 - Improved options for scalability ✅
 - Remove dependence on old infrastructure ✅
-- Deploy ZenHub in the same deployment paradigm as your other applications ✅
+- Deploy Zenhub in the same deployment paradigm as your other applications ✅
 ### 1.3 How do we set up ZHE3?
-- For **ZenHub for Kubernetes**, please refer to the [**k8s-cluster**](https://github.com/ZenHubHQ/zenhub-enterprise/tree/master/k8s-cluster) directory.
-- For **ZenHub as a VM**, please refer to the [**virtual-machine**](https://github.com/ZenHubHQ/zenhub-enterprise/tree/master/virtual-machine) directory.
+- For **Zenhub for Kubernetes**, please refer to the [**k8s-cluster**](https://github.com/ZenhubHQ/zenhub-enterprise/tree/master/k8s-cluster) directory.
+- For **Zenhub as a VM**, please refer to the [**virtual-machine**](https://github.com/ZenhubHQ/zenhub-enterprise/tree/master/virtual-machine) directory.
 ### 1.4 How do we migrate from ZHE2 to ZHE3?
 The *basic* steps to migrate from your existing ZHE2 machine to ZHE3 are:
-1. Decide if you want to deploy **ZenHub for Kubernetes** or **ZenHub as a VM**.
+1. Decide if you want to deploy **Zenhub for Kubernetes** or **Zenhub as a VM**.
 2. Deploy a ZHE3 "landing zone" to prepare for the migration.
 3. Gather data from the existing ZHE2 source instance.
 4. Move the data to a workstation/jump box.
 5. Upload the data to your ZHE3 landing zone.
 
-These steps are explained in much greater detail in [**k8s-cluster**](https://github.com/ZenHubHQ/zenhub-enterprise/tree/master/k8s-cluster) and [**virtual-machine**](https://github.com/ZenHubHQ/zenhub-enterprise/tree/master/virtual-machine).
+These steps are explained in much greater detail in [**k8s-cluster**](https://github.com/ZenhubHQ/zenhub-enterprise/tree/v3.1.2/k8s-cluster) and [**virtual-machine**](https://github.com/ZenhubHQ/zenhub-enterprise/tree/v3.1.2/virtual-machine).
```

Comment:
3.1.2?

Reply:
Yes, the ZHE2->3 instructions are only valid/required for versions lower than ZHE 3.2. So we removed these instructions after this version to avoid confusion. We link directly to that older branch in order for users to see those docs now.